tests/jax/systems/ippo_system_test.py

```diff
@@ -100,20 +95,24 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         nodes_on_gpu=[],
```

Comment:
Do we need to run the evaluator here? Or can we disable it?

Reply:
No it is not necessary if you want we can remove it!